piet-cairo/src/lib.rs

```diff
@@ -113,6 +131,40 @@ impl<'a> RenderContext for CairoRenderContext<'a> {
         self.set_brush(brush);
         self.ctx.stroke();
     }
+
+    fn new_font_by_name(
+        &mut self,
+        name: &str,
+        size: impl RoundInto<Self::Coord>,
+    ) -> Self::FBuilder {
+        CairoFontBuilder {
+            family: name.to_owned(),
+            size: size.round_into(),
+            weight: FontWeight::Normal,
+            slant: FontSlant::Normal,
+        }
+    }
+
+    fn new_text_layout(&mut self, font: &Self::F, text: &str) -> Self::TLBuilder {
+        let text_layout = CairoTextLayout {
+            font: font.0.clone(),
+            text: text.to_owned(),
+        };
+        CairoTextLayoutBuilder(text_layout)
+    }
+
+    fn draw_text(
+        &mut self,
+        layout: &Self::TL,
+        pos: impl RoundInto<Self::Point>,
+        brush: &Self::Brush,
+    ) {
+        self.ctx.set_scaled_font(&layout.font);
+        self.set_brush(brush);
+        let pos = pos.round_into();
+        self.ctx.move_to(pos.x, pos.y);
+        self.ctx.show_text(&layout.text);
```

Comment:
Worth adding a comment that this doesn't shape the text while the other backends do shape the text.

Reply:
Indeed. At some point I'd like to blend in proper text layout, which is then likely to be useful with a number of back-ends, particularly lower level GPU-tuned ones. This is a bit of a hard problem, and I'm not sure what solution is best. But you're certainly right; as long as I'm using the "toy" text API of Cairo, that should be documented.

Reply:
@jrmuizel I added a README, do you think that addresses your concern?

Reply:
Yeah. That looks good. You may want to mention the option of using font-kit to handle font discovery.